packages/plugin-core/src/features/completionProvider.ts

```diff
@@ -109,6 +110,10 @@ export const provideCompletionItems = (
     item.insertText = longRef; //insertText;
 
     item.sortText = padWithZero(index);
+    // Sort notes from current vault before other vaults
+    if (currentVault && !VaultUtils.isEqual(currentVault, note.vault, wsRoot)) {
+      item.sortText = 'x' + item.sortText;
```

Comment:
if i'm reading this correctly, are we updating the sort by adding an `x` in front of the zero padded index? if so, a comment with some details would be great

Reply:
Is this what you had in mind?

Reply:
yep!